mitosheet/mitosheet/code_chunks/step_performers/column_steps/split_text_to_columns_code_chunk.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Saga Inc.
+# Distributed under the terms of the GPL License.
+
+from typing import List
+from mitosheet.code_chunks.code_chunk import CodeChunk
+from mitosheet.sheet_functions.types.utils import is_datetime_dtype, is_string_dtype
+from mitosheet.transpiler.transpile_utils import column_header_list_to_transpiled_code, column_header_to_transpiled_code
+
+
+class SplitTextToColumnsCodeChunk(CodeChunk):
+
+    def get_display_name(self) -> str:
+        return 'Split text to columns'
+    
+    def get_description_comment(self) -> str:
+        sheet_index = self.get_param('sheet_index')
+        column_id = self.get_param('column_id')
+        delimiters = self.get_param('delimiters')
+        column_header = self.post_state.column_ids.get_column_header_by_id(sheet_index, column_id)
+        delimiters_string = (' ,').join(delimiters)
+        return f'Split {column_header} on {delimiters_string}'
+
+    def get_code(self) -> List[str]:
+        sheet_index = self.get_param('sheet_index')
+        column_id = self.get_param('column_id')
+        delimiters = self.get_param('delimiters')
+
+        # The type system says that execution_data can be None, so we need to appease it
+        # TODO: Update the type system so that the execution data is not None when we know it is defined for a step
+        new_column_headers = self.execution_data['new_column_headers'] if self.execution_data is not None else '1' 
```

Comment:
There is a self.get_execution_data function I think! Use this rather than indexing in directly.